### DISCUSSION TITLE
cpp sample

### DISCUSSION THREAD

**Discussion by Hazhir007** [trustwallet/wallet-core#1626] [Q&A]
when compiling and running provided sample (cpp version) the returned sign transaction is as follows:
qoute-> i have only changed the mnemonic btw.

f8498080808080801ca0dc169145fd5784ec5d98c3dea9fc7d5b81af98c15192aec6dc53fa3d3e5f934da00676e4e930ad0bbf5c773c7a3285e80ac1a907fffd078f759639935e118950da

when decoding this raw transaction the provieded data is like so:

{
  "nonce": 0,
  "gasPrice": null,
  "gasLimit": null,
  "to": "0x",
  "value": 0,
  "data": "",
  "from": "0x91995e7843878328d9b539ecfa7f63658f79a53a",
  "r": "dc169145fd5784ec5d98c3dea9fc7d5b81af98c15192aec6dc53fa3d3e5f934d",
  "v": "1c",
  "s": "0676e4e930ad0bbf5c773c7a3285e80ac1a907fffd078f759639935e118950da"
}

which shows gasPrice and gasLimit as null also value as 0 ant toAddress is 0x but the json is not any of those.

any help is appriciated. 
thanks

**Comment by optout21** [comment]
I found the sample code is incorrect, thanks for the note.  Issue tracked in: https://github.com/trustwallet/wallet-core/issues/1627
